src/main/resources/db/migration/V1_14__LOCAL_ADMIN_AUTHORITIES_DATA.sql

```diff
@@ -0,0 +1,162 @@
+Insert into LOCAL_ADMIN_AUTHORITIES (LOCAL_AUTHORITY_CODE,DESCRIPTION,LOCAL_AUTHORITY_CATEGORY,ACTIVE_FLAG,EXPIRY_DATE,CREATE_DATETIME,CREATE_USER_ID,MODIFY_DATETIME,MODIFY_USER_ID,AUDIT_TIMESTAMP,AUDIT_USER_ID,AUDIT_MODULE_NAME,AUDIT_CLIENT_USER_ID,AUDIT_CLIENT_IP_ADDRESS,AUDIT_CLIENT_WORKSTATION_NAME,AUDIT_ADDITIONAL_INFO) values ('BRI','BURE (HMP) LAA','INST','Y',null,to_timestamp('15-JAN-10 14.35.46.371552','DD-MON-RR HH24.MI.SSXFF6'),'OMS_OWNER',null,null,to_timestamp('15-JAN-10 14.35.46.371932','DD-MON-RR HH24.MI.SSXFF6'),'OMS_OWNER','sqlplus@psa10766 (TNS V1-V3)','oracle',null,'pts/4',null);
```

Comment:
taken directly from T3, we could just have subset of prisons but since there is less than 200 we have the whole lot